content/src/content/jcr_root/apps/core/wcm/components/tableOfContent/v1/tableOfContent/_cq_dialog/.content.xml

```diff
@@ -0,0 +1,142 @@
+          jcr:primaryType="nt:unstructured"
+          jcr:title="Table of Content"
+          sling:resourceType="cq/gui/components/authoring/dialog"
+          extraClientlibs="[core.wcm.components.tableOfContent.v1.editor]"
+          helpPath="https://www.adobe.com/go/aem_cmp_table-of-content"
+          trackingFeature="core-components:tableOfContent:v1">
+    <content
+        jcr:primaryType="nt:unstructured"
+        sling:resourceType="granite/ui/components/coral/foundation/container">
+        <items jcr:primaryType="nt:unstructured">
+            <tabs
+                jcr:primaryType="nt:unstructured"
+                sling:resourceType="granite/ui/components/coral/foundation/tabs"
+                maximized="{Boolean}true">
+                <items jcr:primaryType="nt:unstructured">
+                    <tableOfContentItems
+                        jcr:primaryType="nt:unstructured"
+                        jcr:title="Items"
+                        sling:resourceType="granite/ui/components/coral/foundation/container"
+                        margin="{Boolean}true">
+                        <items jcr:primaryType="nt:unstructured">
+                            <columns
+                                jcr:primaryType="nt:unstructured"
+                                sling:resourceType="granite/ui/components/coral/foundation/fixedcolumns"
+                                margin="{Boolean}true">
+                                <items jcr:primaryType="nt:unstructured">
+                                    <column
+                                        jcr:primaryType="nt:unstructured"
+                                        sling:resourceType="granite/ui/components/coral/foundation/container">
+                                        <items jcr:primaryType="nt:unstructured">
+                                            <title
+                                                jcr:primaryType="nt:unstructured"
+                                                sling:resourceType="granite/ui/components/coral/foundation/form/textfield"
+                                                fieldDescription="Leave empty to use the page title."
+                                                fieldLabel="Title"
+                                                name="./jcr:title"/>
+                                            <tableOfContentItems
+                                                jcr:primaryType="nt:unstructured"
+                                                sling:resourceType="core/wcm/components/commons/editor/dialog/childreneditor/v1/childreneditor"/>
+                                        </items>
+                                    </column>
+                                </items>
+                            </columns>
+                        </items>
+                    </tableOfContentItems>
+                    <properties
```

Comment:
This whole tab is messy. I don't think this component needs more than one *Properties* tab, which groups all the options:
* title
* heading
* ID
* [maybe] list type (ordered/unordered)
* [maybe] threshold for collecting titles from page (ie: >=h3)